packages/ui/api/ai.ts

```diff
@@ -49,7 +36,7 @@ export default async (req: Request) => {
     ];
 
     const response = await openai.chat.completions.create({
-      model: 'gpt-4-1106-preview',
+      model: getEnvironmentVariableOr('OPEN_AI_MODEL', 'gpt-4o-mini'),
```

Comment:
For consistency with `OPENAI_API_KEY` name format:
```suggestion
      model: getEnvironmentVariableOr('OPENAI_MODEL', 'gpt-4o-mini'),
```